docs/isr.md

```diff
@@ -21,6 +21,16 @@ request is made for stale content, the page will be regenerated in the backgroun
 but it can take up to 60 seconds before the new content is then updated in all CDN nodes if they already had a cached
 copy.
 
+If the static regeneration relies on local files in your repository they need to be bundled with the handler functions. 
+This can be done by modifying your [file based configuration](https://docs.netlify.com/configure-builds/file-based-configuration).
+An entry to the `included_files` option needs to be added under the `functions` option. 
```

Comment:
```suggestion
An entry to the `included_files` option needs to be added under the `functions` option. You should be careful to not include unnecessary files, particularly large files such as images or videos, because there is a 50MB size limit for each handler function.
```